lib/txt/news

```diff
@@ -1,3 +1,17 @@
+09-03-21 : Encamp and Garrotte Moved
+           - Encamp and garrotte have accidentally been in the wrong disciplines for
+           a long time. This was causing some memory leakage on the server, which was
+           fixed by moving the skills to their originally intended disciplines.
+
+           - Encamp is now in the Advanced Adventuring discipline instead of Adventuring
```

Comment:
Hey, encamp was moved to regular adventuring to make it easy for folks to hop in, kill a couple of mobs, and not have to slog all the way to the inn.

Reply:
Reversed this in https://github.com/sneezymud/sneezymud/pull/380/commits/3b7b46ed197a2ae51b84b707b8684d776b44a00e. Encamp is now fully moved to the basic adventuring disc, which resolves the memory leak just as well.